pkg/controller/context.go

```diff
@@ -149,6 +149,9 @@ type CertificateOptions struct {
 	// EnableOwnerRef controls whether the certificate is configured as an owner of
 	// secret where the effective TLS certificate is stored.
 	EnableOwnerRef bool
+	// CopiedAnnotationPrefixes defines which annotations should be copied
+	// Certificate -> CertificateRequest, CertificateRequest -> Order.
+	CopiedAnnotationPrefixes []string
```

Comment:
We should also extend this for `CertificateSigningRequest -> Order`

https://github.com/jetstack/cert-manager/blob/f278903c4eac53ccfd4a4543f99553142a86f71d/pkg/controller/certificatesigningrequests/acme/acme.go#L301

Reply:
I didn't actually explicitly add this to `CertificateRequest` -> `Order` because I assumed that, most of the time, `CertificateRequest`s get created from `Certificate`s and if not then probably they anyway wouldn't be created by GitOps tools or anything that manages resources long term using these kind of annotations.
Somewhat unsure about whether these kind of annotations would matter for `CertificateSigningRequest`s although theoretically there's no harm in filtering them since there wouldn't be that many anyway.